src/app/utils/miners.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  isHexString,
+  getNumber,
+  randomBytes,
+  keccak256,
+  toBeHex,
+  toBigInt,
+  ethers,
+} from "ethers";
+
+export default class Miner {
+  public static MAX_NUMBER = ethers.MaxUint256;
```

Comment:
I think this is the only place ethers is being used. We can use viem for this value as well and get rid of ethers on the repo so that we don't have to mix dependencies.